Formula/gopass.rb

```diff
@@ -0,0 +1,43 @@
+class Gopass < Formula
+  desc "The slightly more awesome Standard Unix Password Manager for Teams"
+  homepage "https://www.justwatch.com/gopass"
+  url "https://www.justwatch.com/gopass/releases/1.2.0/gopass-1.2.0-darwin-amd64.tar.gz"
```

Comment:
Ah, of course it will. That's a bit of a brain fail there on my part. I'll get that fixed up